load_balancer.tf

```diff
@@ -0,0 +1,82 @@
+resource "azurerm_lb" "scale_set_lb" {
+  location            = var.location
+  name                = var.load_balancer_name
+  resource_group_name = var.resource_group_name
+  sku                 = "Standard"
+
+  frontend_ip_configuration {
+    name      = "corelight-sensor-lb-ip"
+    subnet_id = azurerm_subnet.subnet.id
+  }
+
+  tags = var.tags
+}
+
+resource "azurerm_lb_backend_address_pool" "management_pool" {
+  loadbalancer_id = azurerm_lb.scale_set_lb.id
+  name            = "management-pool"
+}
+
+resource "azurerm_lb_backend_address_pool" "monitoring_pool" {
+  loadbalancer_id = azurerm_lb.scale_set_lb.id
+  name            = "monitoring-pool"
+}
+
+resource "azurerm_lb_probe" "sensor_health_check_probe" {
+  loadbalancer_id     = azurerm_lb.scale_set_lb.id
+  name                = "health-check"
+  port                = 443
+  request_path        = "/api/system/healthcheck/"
+  protocol            = "Https"
+  interval_in_seconds = 30
+  probe_threshold     = 3
+}
+
+resource "azurerm_lb_rule" "monitoring_vxlan_lb_rule" {
+  loadbalancer_id                = azurerm_lb.scale_set_lb.id
+  name                           = "vxlan-lb-rule"
```

Comment:
Resource name is hard coded, should this be a var?

Reply:
All the resources that are somewhat "hidden" as internal configs in other resources have hard coded names. I'll pull them out to variables with defaults though for consistency sake.